lms/static/styles/components/_BasicLtiLaunchApp.scss

```diff
@@ -1,19 +1,18 @@
-.BasicLtiLaunchApp__form {
-  margin-top: 10vh;
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+.BasicLtiLaunchApp__dialog-container {
+  /* This is a hack to force the height. */
+  /* In cases where multiple children of the iframe */
+  /* have height > 0, and one of the children has */
+  /* height=100%, the height of all the children */
+  /* exceeds the height of the iframe. This could be */
+  /* fixed by adding flex to the iframe but since */
+  /* we cannot modify the iframe styling, we have to */
+  /* hardcode the height instead. */
+  height: 80vh;
   position: relative;
-
-  margin-left: auto;
-  margin-right: auto;
-  max-width: 500px;
-  width: fit-content;
 }
 
-.BasicLtiLaunchApp__button {
-  margin-top: 20px;
-  align-self: flex-end;
+.BasicLtiLaunchApp__dialog {
+  width: 75%;
```

Comment:
Adding `max-width: 500px` here would make these dialogs a little more consistent in appearance with other dialogs in the app.